hydrus/data/crud.py

```diff
@@ -56,6 +56,10 @@ def insert(object_, id_=None):
     """Insert an object to database [POST] and returns the inserted object."""
     # NOTE: We are inserting the object, no need to check if similar one already exists.
     #       Data can be redundant/identical, they must have different "@id"
+
+    # TODO: @chrizandr You're not setting instance.type_ and you're not using keymap
+    # to dereference class types. Try any other category other then thermal for insertion
+    # Also, make sure crud.py passes all the tests in test_db.py (Currently all failing)

```

Comment:
@chrizandr I know we have `type` attribute in our Graph but that is for what type of graph it is. For example `III` or IAI` etc. But the type attribute in `instance` is for which `class` the object belongs to. For example `Spacecraft_power` or `Spacecraft_CDC` etc. The both are entirely different cases and are needed to be defined separately.